sdks/python/build.gradle

```diff
@@ -210,7 +210,7 @@ task portableWordCount {
 def portableWordCountTask(name, streaming, crossLanguage) {
   tasks.create(name) {
     dependsOn = ['installGcpTest']
-    mustRunAfter = [':beam-runners-flink_2.11-job-server-container:docker', ':beam-sdks-python-container:docker', ':beam-sdks-java-container:docker', ':beam-runners-core-construction-java:buildTestExpansionServiceJar']
+    mustRunAfter = [':beam-runners-flink_2.11-job-server-container:docker', ':beam-sdks-python-container:docker', ':beam-sdks-java-container:docker', ':beam-runners-java-fn-execution:buildTestExpansionServiceJar']
```

Comment:
We can address that separately, but I don't think this task should be tied to building the java container or expansion service jar. If the expansion service is a job server dependency, isn't that redundant here anyways?

Reply:
Yes, can address this next. It is actually still required because the `TestExpansionService` builds a new jar which contains the test dependencies which would normally not be part of the jar. I did not change the test setup for now.